ports/mpmcqueue/portfile.cmake

```
@@ -3,15 +3,14 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO rigtorp/MPMCQueue
-    REF 5883e32b07e8a60c22d532d9120ea5c11348aea9
-    SHA512 4adbbe5e014e0ef5c7030aaa9faa4e07e2c65753cd89c770da250811c13776576c4f1caf4144542318c41ebc7433b106e802c482a5d44572963a5ab59047257e
+    REF 28d05c021d68fc5280b593329d1982ed02f9d7b3
+    SHA512 e3305ecac05d48814d75adcb85fa165eec3a439a17dd99f8b0d2c095e40b2f98bd4bcf167cf8268f84d09aa172ab66b30573d9d3ad4908c10dc5bec632529b8a
     HEAD_REF master
 )
 
-file(COPY
-    ${SOURCE_PATH}/include/rigtorp/MPMCQueue.h
-    DESTINATION ${CURRENT_PACKAGES_DIR}/include/rigtorp
-)
+vcpkg_cmake_configure(SOURCE_PATH "${SOURCE_PATH}")
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup()
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/MPMCQueue)

file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib") 
```
The method `vcpkg_cmake_config_fixup` is used to merge release and debug CMake targets and configs to support multiconfig generators.
Referance: https://github.com/microsoft/vcpkg/blob/master/ports/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake